os/board/esp32_devkit/src/esp32_appinit.c

```diff
@@ -94,7 +94,7 @@
  *
  ****************************************************************************/
 
-int board_app_initialize(uintptr_t arg)
```

Comment:
please also change this function Description comment accordingly.

Reply:
Modified.